### ISSUE TITLE
BrowserStack Feature Detection Issues on IE

### ISSUE THREAD

**Issue by kitsonk** [theintern/leadfoot#78]
Using the version of Leadfoot with Intern 3.2.0 and the following configuration:

```js
{ browser: 'IE', 'browser_version': '10.0', 'os': 'Windows', 'os_version': '8' },
{ browser: 'IE', 'browser_version': '11.0', 'os': 'Windows', 'os_version': '10' }
```

I am getting what appears to be an errors during the detection of the Selenium features which freezes the session:

* [IE10](https://www.browserstack.com/automate/builds/1899a5b01c40f3dfbd8e6195f511c5a7b911b213/sessions/4e58ec4ea31c34b7d293b0fb381f7bafcfd65272#automate_button)
* [IE 11](https://www.browserstack.com/automate/builds/1899a5b01c40f3dfbd8e6195f511c5a7b911b213/sessions/1f0b409a38f6ec1d60f7dc1e00bc1b101333ca5b#automate_button)

**Comment by jason0x43** [comment]
Those example links aren't working for me (BS says they don't exist). What project are you seeing issues with?

I've run the dojo-core suite several times on BS and I'm not seeing any issues with IE 10 or 11. Note that error messages during feature testing are normal; a Selenium error generally indicates a failed test (and a missing/broken feature).

**Comment by kitsonk** [comment]
Apologies, it is my own BrowserStack account and didn't realise they weren't shareable.

It is [dojo/compose](https://github.com/dojo/compose/tree/feature-update-intern).

Here is a screenshot of where it gets hung up:

<img width="485" alt="run_selenium_tests_in_300__desktop_and_mobile_browsers" src="https://cloud.githubusercontent.com/assets/1282577/15114118/3b9c6e4e-15f0-11e6-88c6-019c5c664427.png">

**Comment by jason0x43** [comment]
Ah, I can see how that test breaking would cause an issue. However, I'm curious as to why it's working for me but not for you. I'll have to try compose.

Just out of curiosity, what kind of host are you running the tests from?

**Comment by kitsonk** [comment]
I was running them from my Mac (OSX 10.11.5 Beta) and NodeJS v6.0.0.

**Comment by tomodian** [comment]
@jason0x43 having the same situation, you can see the video log here https://youtu.be/ymvn4WSfJes and text log is reported at `00:28`.

```
Command not found: GET /session/0d92a737-292b-4703-96b0-4e02907ebcf2/log/types 
Command duration or timeout: 2 milliseconds 
Build info: version: '2.51.0', revision: '1af067d', time: '2016-02-05 19:15:17' 
System info: host: '185-***', ip: '185.***', os.name: 'windows', os.arch: 'x86', os.version: '6.3', java.version: '1.8.0_65' 
Driver info: org.openqa.selenium.ie.InternetExplorerDriver 
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=false, handlesAlerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=true, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=about:blank, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=dismiss}] 
```

**Comment by jason0x43** [comment]
This is fixed in 1.6.8 (Intern 3.2.1); at least, the offending defect check has been disabled for now.

**Comment by kitsonk** [comment]
Yes, no longer an issue.
